<launch>
	<node pkg="trajectory_reshaping_adaptive_view" type="trajectory_reshaping_adaptive_view_node" name="trajectory_reshaping_adaptive_view_node" output="screen">
	<rosparam file="$(find trajectory_reshaping_adaptive_view)/config/trajectory_reshaping_adaptive_view.yaml" command="load" />
	</node>
</launch>
